examples/auth-ssl/certs/cert.pem

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Since certs have an expiration date, I would suggest not checking them in and either leave it as an exercise for the user running the example to create it with the instructions you provided or use one of those npm modules that auto generate them.

Reply:
This seems to be a good point. I will remove the certs and ask them to be generated in the Readme.md.